src/cache/cache_las.c

```diff
@@ -1113,8 +1113,8 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
 		 * Remove entries from the lookaside that have aged out and are
 		 * now no longer needed.
 		 */
-		WT_ERR(cursor->get_value(cursor,
-		    &las_txnid, &las_timestamp, &upd_type, &las_value));
+		WT_ERR(cursor->get_value(cursor, &las_txnid,
+		    &las_timestamp, &upd_type, &las_value, &prepare_state));
```

Comment:
we can skip the prepared updates as part of sweep.  Below call to __wt_txn_visible_all will return "True" for prepared updates.  add this condition also to below if.